### ISSUE TITLE
IcedFormatAll throws error

### ISSUE THREAD

**Issue by uosl** [liquidz/vim-iced#172]
Using `IcedFormat` works fine, but `IcedFormatAll` prints the following error.

```
Unexpected error: 'vital: Async.Promise: Timeout'                                                                                                                                                                                  
Error detected while processing function iced#format#all:
line   15:
E715: Dictionary required
Press ENTER or type command to continue
```

Tested with neovim v0.4.3 in a CLJS file where I had previously `:IcedJackIn` and `:IcedStartCljsRepl figwheel-sidecar`.

**Comment by liquidz** [comment]
@uosl Thanks for your reporting!
The problem is caused by the following sync process.
https://github.com/liquidz/vim-iced/blob/0.13.0/autoload/iced/format.vim#L30

I'll fix it tomorrow!

**Comment by liquidz** [comment]
@uosl I fixed the problem at `dev` branch.
Could you try `dev` branch?

**Comment by uosl** [comment]
Thanks for the quick fix!

Oddly enough, I wasn't able to reproduce the issue on my home computer. I'll try to reproduce it on the machine I experienced it on monday, so I can test the fix properly.

**Comment by uosl** [comment]
I didn't manage to reproduce it where I experienced it earlier, so I guess a restart fixed whatever oddities in the environment that caused it. \\=

I did try the `dev` branch and `IcedFormatAll` worked there as well.

Thanks again for looking into this!

**Comment by liquidz** [comment]
@uosl This fix has been released at #173
